pkg/core/runtime/runtime.go

```diff
@@ -62,7 +62,7 @@ type RuntimeContext interface {
 	Metrics() metrics.Metrics
 	EventReaderFactory() events.ListenerFactory
 	APIInstaller() api_server.APIInstaller
-	XDSHooks() *xds_hooks.Hooks
+	XDSHooks() []xds_hooks.ResourceSetHook
```

Comment:
I'd prefer to keep it as is. I actually use this object in the external plugin PoC (another type of hooks).